requirements_all.txt

```diff
@@ -543,3 +543,7 @@ yahooweather==0.8
 
 # homeassistant.components.zeroconf
 zeroconf==0.17.6
+
+# homeassistant.components.sensehat
+# homeassistant.components.sensor.sensehat
+sensehat
```

Comment:
I added "sesne-hat" dependency both in requirement file and in docker, but I think I miss something....